src/sagemaker/jumpstart/types.py

```diff
@@ -65,7 +65,7 @@ def __str__(self) -> str:
         {'content_bucket': 'bucket', 'region_name': 'us-west-2'}"
         """
 
-        att_dict = {att: getattr(self, att) for att in self.__slots__}
+        att_dict = {att: getattr(self, att) for att in self.__slots__ if hasattr(self, att)}
```

Comment:
curious: why do you need this now?

Reply:
this bug got exposed after i fixed a unit test